examples/fields/country.ts

```diff
@@ -0,0 +1,274 @@
+import { OptionField } from '@flatfile/configure'
+
+/**
+ * An OptionField that provides a list of countries
+ * 
+ * @return {Bbject} an option field
+*/
+export default function Country(): object {
+  // convert countries array to map object for OptionField format
+  const countriesMap = new Map(
+    countryList.map(function(value) {
+      const key = value.toLowerCase().replace(' ', '-')
+      return [key, value]
+    })
+  )
+
+  return OptionField({
+      label: 'Country',
+      options: countriesMap,
+      description: "Pick a country"
+  })
+}
+
+const countryList = [
+  "Afghanistan",
+  "Åland Islands",
+  "Albania",
+  "Algeria",
+  "American Samoa",
+  "Andorra",
+  "Angola",
```

Comment:
Take a look at https://github.com/FlatFilers/platform-sdk-mono/blob/main/packages/configure/src/stdlib/RegionCast.ts

and 
https://github.com/FlatFilers/platform-sdk-mono/blob/main/packages/configure/src/stdlib/SubstitutionCast.ts

Reply:
Can we just remove this for now, or properly use countrycast?